modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -137,7 +138,9 @@ private void _validate(ObjectEntry objectEntry) {
 		try {
 			Map<String, Serializable> values = objectEntry.getValues();
 
-			Matcher matcher = _pathPattern.matcher((String)values.get("path"));
+			String pathString = (String)values.get("path");
+
+			Matcher matcher = _pathPattern.matcher(pathString);
```

Comment:
AFAIK, the `^[/]` from the pattern could be replaced just with `/` for two reasons:

- The `^` is useless when we use `matches` method
- One unique char could be used without brackets

Please test it